labs-ios-starter/Model/Profile.swift

```diff
@@ -10,9 +10,9 @@ import UIKit
 
 struct Profile: Codable {
     
-    let id: String
-    let email: String
-    let name: String
+    let id: String?
+    let email: String?
+    let name: String?
```

Comment:
Remove the optionality here. I only told you to make them optional for the sample endpoint, but your deployed endpoint shouldn't allow any of these to be nil

Reply:
Ok, will do.